src/elm/Page/Community/Settings/Features.elm

```diff
@@ -68,7 +77,7 @@ view loggedIn model =
         , div
             [ class "container w-full divide-y"
             ]
-            [ toggleView translations (translate "objectives.title_plural") model.hasActions ToggleActions "actions"
+            [ toggleView translations (translate "community.objectives.title_plural") model.hasObjectives ToggleActions "actions"
```

Comment:
I agree. It was originally named `hasActions`, but when doing the contract migration @lucca65 chose to use `hasObjectives`. Since it's a bit of a pain to make these changes on the blockchain, I decided to change the frontend to use `hasObjectives` instead of changing the contract to use `hasActions`.

That leaves the backend with the old state of `hasActions`. I'll open an issue and mark this PR as blocked for a little bit while I fix it. Thanks for pointing it out :)

Reply:
Hey maybe i’ve contributed to the confusion here. I didn’t remember what was the name on the front/back

the reason behind choosing objectives over actions is that the main idea of this feature is having a track able way of having communities layout their objectives and know how to reach them. No action can exist outside of an objective. The ideia is to grow this to follow a guideline called “Value based decision making” you can find some material on this methodology on the web (i can share an ebook too if you are interested)

We still didn’t had the chance to discuss more about Cambiatus reasoning and theoretical background, but we have some solid and different  materials that we use as base for being the way we are.

Values of a community is more broadly shown by their objectives, actions are just a way of reaching it. Since we can have ineffective actions (that is actions that don’t help you reach your objective) it’s better to call objectives and not actions — since the final thing we want is not to do actions, but reach the community objective 

this is still in its early stages as a decision making and value proposition management in a community.

Reply:
Fixed by cambiatus/backend/pull/48